plugins/content/createmenulink/createmenulink.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension version="3.6" type="plugin" group="content" method="upgrade">
+    <name>PLG_CONTENT_CREATEMENULINK</name>
+    <author>Joomla! Project</author>
+    <creationDate>July 2016</creationDate>
+    <copyright>Copyright (C) 2005 - 2016 Open Source Matters. All rights reserved.</copyright>
+    <license>GNU General Public License version 2 or later; see LICENSE.txt</license>
+    <authorEmail>admin@joomla.org</authorEmail>
+    <authorUrl>www.joomla.org</authorUrl>
+    <version>1.1.0</version>
+    <description>PLG_CONTENT_CREATEMENULINK_XML_DESCRIPTION</description>
+    <files>
+        <filename plugin="createmenulink">createmenulink.php</filename>
+	    <folder>forms</folder>
+    </files>
+	<config>
+		<fields name="params">
+			<fieldset name="basic">
+				<field
+					name="componentview"
```

Comment:
Sorry, thought you know - Code Style. In this case your formatting here is not correct :-)